library/classes/Prescription.class.php

```diff
@@ -297,16 +297,14 @@ function get_refills() {
     }
 
     function set_size($size) {
-        if (is_numeric($size)) { $this->size = $size; }
+        $this->size = $size
```

Comment:
change this to
`$this->size = preg_replace("/[^0-9\/\.]/", "",$size);`
It allows numbers, periods, fowardslash and backslash.